.azure-pipelines/util/get-github-token-steps.yml

```diff
@@ -0,0 +1,12 @@
+steps:
+- task: PowerShell@2
+  displayName: Get GitHub Token
+  inputs:
+    targetType: 'inline'
+    script: |
+      $httpConfigs = (git config --get-regexp "http" )
+      $authConfig = $httpConfigs | Where-Object {$_ -like "*AUTHORIZATION*"}
+      $token = ($authConfig -split ' ')[-1]
+      $ghToken = ([System.Text.Encoding]::ASCII.GetString([System.Convert]::FromBase64String($token)) -split ':')[1]
```

Comment:
How comfortable are we building our pipeline upon this low level technical detail?

Reply:
Although it looks a bit tricky, it's just a general git feature used by Azure Pipelines.  I think it won't change for years. Even if it does change, we can use PAT for a workaround immediately. (I think it's stable because if it's changed, it will be a big breaking change of Git or GitHub.)